src/api/UserApi.ts

```diff
@@ -18,7 +27,6 @@ export class UserApi extends Controller {
     ): Promise<Response<User>> {
         const token = request.headers['authorization']
         const response = await UserService.verifyAndCreateUser(token)
-        if(response.data) this.setStatus(201)
```

Comment:
Czemu to usunąłeś? Status i tak ustawia się na 201 czy jest jakiś inny tego powód?

Reply:
Głównie dlatego, że teraz serwis zwraca usera nie tylko w momencie utworzenia, ale również w momencie weryfikacji. 

Ogólnie ten endpoint służy zarówno do logowania jak i "rejestracji". Dane były zwracane tylko przy pierwszym logowaniu (rejestracji), a przy kolejnych zwracany był `null`. Na tej podstawie robiony był ten check i zwracane 201. O ile 201 przy tworzeniu usera w bazie jak najbardziej ma sens, o tyle zwracanie nulla za każdym kolejnym logowaniem ma tego sensu mniej. Teraz mógłbym sprawdzać w jakiś sposób, czy jest to 1 czy kolejne logowanie i na tej podstawie zwracać 201/200, ale nie daje to żadnej wartości w późniejszym użyciu, więc po prostu użyłem kodu bardziej generycznego i takiego, który zdecydowanie częściej wystąpi :D